### ISSUE TITLE
Error saving a patch with 'FaceColor' as 'none'

### ISSUE THREAD

**Issue by philipptempel** [matlab2tikz/matlab2tikz#739]
Trying to export a 3D figure with a patch that has 'FaceColor' set to none, I'm getting the following error

```matlab
Error using matlab2tikz>patchcolor2xcolor (line 4305)
Color model 'none' not allowed here. Make sure this case gets intercepted
before.
Error in matlab2tikz>getColor (line 4238)
                [m2t, xcolor] = patchcolor2xcolor(m2t, color, handle);
Error in matlab2tikz>setColorsOfPatches (line 1960)
        [m2t,xFaceColor] = getColor(m2t, handle, s.faceColor, 'patch');
Error in matlab2tikz>drawPatch (line 1877)
        [m2t, drawOptions, Vertices, Faces, verticesTableOptions, ptType, ...
Error in matlab2tikz>handleAllChildren (line 646)
                [m2t, str] = drawPatch(m2t, child);
Error in matlab2tikz>drawAxes (line 852)
    [m2t, childrenEnvs] = handleAllChildren(m2t, handle);
Error in matlab2tikz>saveToFile (line 450)
        m2t = drawAxes(m2t, relevantAxesHandle);
Error in matlab2tikz (line 352)
saveToFile(m2t, fid, fileWasOpen);
Error in mwe (line 23)
matlab2tikz('test.tikz', 'ShowInfo', false); 
```

An MWE would be

```matlab
vertices = [...
-8.5448047                -5.5575856                 -0.589861; ...
  8.224472                -5.5575856                 -0.589861; ...
  8.224472                 6.4639869                 -0.589861; ...
-8.5448047                 6.4639869                 -0.589861; ...
-8.5448047                -5.5575856                 3.2502162; ...
  8.224472                -5.5575856                 3.2502162; ...
  8.224472                 6.4639869                 3.2502162; ...
-8.5448047                 6.4639869                 3.2502162; ...
];

faces = [...
     1     2     3     4; ...
     5     6     7     8; ...
     1     2     6     5; ...
     2     3     7     6; ...
     3     4     8     7; ...
     4     1     5     8; ...
 ];

figure;
% For the sake of an example, force a 3D plot (even though this does not change the error code produced)
plot3(NaN, NaN, NaN);
patch('Vertices', vertices, 'Faces', faces, 'FaceColor', 'none');

drawnow;

matlab2tikz('test.tikz', 'Height', '\figureheight', 'Width', '\figurewidth', 'ShowInfo', false);
```

If I substitute ```'FaceColor'``` ```'none'``` by ```'r'``` it works fine. Using a struct as the first argument to patch() doesn't change anything.

Running Matlab r2014b and the most recent HEAD of matlab2tikz (matlab2tikz@3a1ee10dee). Error reproducible with develop (matlab2tikz@892a4ec100), but not in 0.6.0 (matlab2tikz@adace022dd).

I think this issue is related to #174 but since that's quite old, I thought of opening a new issue

**Comment by egeerardyn** [comment]
I can confirm this behavior in R2014a and R2014b using the latest `develop` branch.

**Comment by egeerardyn** [comment]
For your MWE, you can seemingly work around the issue by replacing the part that fails, i.e. 
```matlab
case 'none'
     error('matlab2tikz:anycolor2rgb:ColorModelNoneNotAllowed',...
           ['Color model ''none'' not allowed here. ',...
            'Make sure this case gets intercepted before.']);
```
with
```matlab
case 'none'
    xcolor = 'none';
```

We will incorporate this into the main code in the near future, but I first want to see why that error message was placed there in the first place.

**Comment by philipptempel** [comment]
Thanks @egeerardyn, that was an amazingly quick response and workaround. Now I can export my 3d plots nicely into TikZ.

**Comment by PeterPablo** [comment]
Fixed by #740
